index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
```

Comment:
[OPTIONAL] Please consider adding a `title` to your project. Making this habit would make you a better dev. 😃😃😃